doc/source/mutest.rst

```diff
@@ -119,5 +119,88 @@ used to check that the state of the interface actually changes.
   step("r1", 'vtysh -c "conf t\n interface eth0\n shut"')
   match_step("r1", 'vtysh -c "show interface eth0", "DOWN", "Check for interface DOWN")
 
+An additional json variant of various functions are also available, namely
+:py:func:`match_step_json`, :py:func:`wait_step_json`, and :py:func:`step_json`.
+These functions act similarly to their base variants, however, instead of
+performing a regular expression search, they will pass if the json output of
+``cmd`` matches the provided expected json (``match``). By default, two json
+successfully match if the following rules hold true:
+
```

Comment:
Additionally, json variants of the above functions are also available, namely
:py:func:`match_step_json`, :py:func:`wait_step_json`, and :py:func:`step_json`.
These json variants compare a json value ``match`` with the json result from ``cmd``.

By default a comparison succeeds if ``match`` is a subset of the ``cmd``result
(i.e., all data within ``match`` is present in the ``cmd`` result). By example the
comparison succeeds when the following rules hold true: